portfolio/src/main/webapp/index.html

```diff
@@ -46,7 +46,63 @@ <h2>About Me </h2>
 <div id="timelapse">
   <h2>Timelapses</h2>
   <p>
-    Here are some videos that I have taken and edited
+    Here are some videos that I have taken and edited.
+  <div id="timelapse-slideshow" class="carousel slide" data-ride="carousel">
+    <ol class="carousel-indicators">
+      <li data-target="#timelapse-slideshow" data-slide-to="0" class="active"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="1"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="3"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="4"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="5"></li>
+      <li data-target="#timelapse-slideshow" data-slide-to="6"></li>
+    </ol>
+    <div class="carousel-inner">
+      <div class="carousel-item active">
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/1.mov" type="video/mp4" />
+        </video>
+        Sorry the embedded videos is not supported by your browswer.
+      </div>
+      <div class="carousel-item">
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/2.mov" type="video/mp4" />
+        </video>
+      </div>
+      <div class="carousel-item">
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/3.mov" type="video/mp4" />
+        </video>
+        Sorry the embedded videos is not supported by your browswer.
+      </div>
+      <div class="carousel-item">
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/4.mov" type="video/mp4" />
+        </video>
+        Sorry the embedded videos is not supported by your browswer.
+      </div>
+      <div class="carousel-item">
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/5.mov" type="video/mp4" />
+        </video>
+        Sorry the embedded videos is not supported by your browswer.
+      </div>
+      <div class="carousel-item">
+        <video class="video-fluid" autoplay loop muted>
+          <source src="./resources/videos/6.mov" type="video/mp4" />
+        </video>
+      </div>
+    </div>
+    <a class="carousel-control-prev p-3 mb-2 bg-dark text-white" href="#timelapse-slideshow" role="button"
+      data-slide="prev">
```

Comment:
Indent 4 spaces when attributes wrap to the next line. See https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

Reply:
Made Changes